test/langtools/tools/javac/valhalla/lworld-values/CheckFinal.out

```diff
@@ -2,6 +2,4 @@ CheckFinal.java:14:13: compiler.err.cant.assign.val.to.final.var: fi
 CheckFinal.java:15:13: compiler.err.cant.assign.val.to.final.var: fe
 CheckFinal.java:17:13: compiler.err.cant.assign.val.to.final.var: xsf
 CheckFinal.java:19:29: compiler.err.cant.inherit.from.final: CheckFinal
-CheckFinal.java:19:42: compiler.err.concrete.supertype.for.primitive.class: compiler.misc.anonymous.class: CheckFinal$1, CheckFinal
-CheckFinal.java:19:25: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: compiler.misc.anonymous.class: CheckFinal, CheckFinal)
-6 errors
+4 errors
```

Comment:
The (now deleted) attempt to avoid further secondary errors in Attr.java having outlived its purpose was actually causing additional errors to be emitted! Fixed.